pandas/conftest.py

```diff
@@ -1146,6 +1146,8 @@ def nullable_string_dtype(request):
     * 'string'
     * 'arrow_string'
     """
+    from pandas.core.arrays.string_arrow import ArrowStringDtype  # noqa: F401
```

Comment:
I've added this to force the registration of the `arrow_string` dtype. running some tests individually sometimes fails. will be removed in #39908

Reply:
you need to handle the import error here and xfail

Reply:
can you elaborate or I can just remove.

Reply:
this would raise an import error right on some builds? if so simply xfail if this errors and the tests would also xfail

that way u can use this fixture w/o worrying too much

Reply:
The tests are skipped with the fixture. It's passing tests that may fail if the `arrow_string` dtype is not registered, it's not part of the public api yet. and won't be since we are going to use the parameterized dtype in #39908 instead.

Reply:
see also https://github.com/pandas-dev/pandas/pull/39908#discussion_r585581525

Reply:
> this would raise an import error right on some builds?

no. importing the ArrowStringDtype is safe.

Reply:
i c, ok then